code/modules/food_and_drinks/machinery/processor.dm

```diff
@@ -24,6 +24,8 @@
 	 * This allows for different types of processor to produce different outputs from same input as long as the recipes require different processors.
 	 */
 	var/static/list/processor_inputs
+	/// Awkward variable for examining
+	var/slime_time = FALSE
```

Comment:
Instead of this I feel you could just make it more clear in the description that the slime processor is specialized for that use.